range_queries/persistent_segment_tree_with_lazy_propagation.cpp

```diff
@@ -0,0 +1,243 @@
+/**
+ * @file
+ * @brief [Persistent segment tree with range updates (lazy propagation)](https://en.wikipedia.org/wiki/Persistent_data_structure - https://www.geeksforgeeks.org/lazy-propagation-in-segment-tree/)
```

Comment:
This is Markdown, and it won't work properly if you add two links like you did. Please choose one.